src/app/core/services/geojson/geojson.service.ts

```diff
@@ -15,10 +15,18 @@ const DEBUG_TAG = 'GeoJSON';
 export class GeoJSONService {
   private db = inject(DatabaseService);
   private logger = inject(LoggingService);
+  private initialized = false;
+  private changedMetadataItem = new Subject<GeoJSONItem>();
+  private removedMetadataItemId = new Subject<string>();
 
+  /** Metadata for alle lagrede spor */
   metadata = signal<GeoJSONItem[]>([]);
```

Comment:
Denne burde kanskje gjøres privat og heller eksponere en versjon av den som asReadonly (eller hva det heter), siden oppdateringer av den nå bør gjøres direkte gjennom updateMetadata. 
Det er ikke så veldig farlig siden bare vi selv er brukere av servicen, men du kan vurdere det.

Det er jo lett å oppdatere metadata uten at this.changedMetadataItem.next(item) kalles sånn som servicen er nå.